openfisca_france/parameters/impot_revenu/calcul_revenus_imposables/abat_rni/enfant_marie/montant.yaml

```
@@ -88,6 +88,8 @@ values:
     value: 5947
   2020-01-01:
     value: 5959
+  2022-01-01: 
```

Ah oui, c'est moi qui ai fait l'erreur !
```suggestion
  2021-01-01: 
```